### PR TITLE
Deduplicate potentialLocations results

### LINKED ISSUE #576: The same potential location of participants.json listed twice
Context: https://neurostars.org/t/tabular-file-contains-custom-columns-not-described-in-a-data-dictionary/2513

Is:
`Evidence:	Columns:  study_id not defined, please define in: /participants.json, /participants.json`

Should be:
`Evidence:	Columns:  study_id not defined, please define in: /participants.json`

### PR DESCRIPTION
**Pull request by bGrass**
Fixes #576 

also fixes testing bugs caused by renaming of INCF organization on GitHub

### CODE REVIEW

File: utils/files/potentialLocations.js [hunk 1]
```diff
@@ -15,8 +15,12 @@ const potentialLocations = path => {
   }
 
   // generate relevant paths and put into closest -> root order
-  let paths = [path].concat(potentialPaths(components)).reverse()
-  return paths
+  const potentials = potentialPaths(components)
+  if (path != potentials) {
```
Comment (chrisgorgo) [inline utils/files/potentialLocations.js]: Isn't `potentials` always a list and therefore this statement would never be true?

**Review by olgn — APPROVED** [review]

**Comment by olgn** [comment]
@bGrass this looks good - perhaps a unit test is in order for this edge case?

**Comment by bGrass** [comment]
> @bGrass this looks good - perhaps a unit test is in order for this edge case?

@olgn How's this look?

**Review by olgn — APPROVED** [review]
